cmd/analyse.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/fabric8-analytics/cli-tools/analyses/driver"
+	sa "github.com/fabric8-analytics/cli-tools/analyses/stackanalyses"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var manifestFile string
+
+// analyseCmd represents the analyse command
+var analyseCmd = &cobra.Command{
+	Use:     "analyse",
+	Short:   "analyse performs Stack Analyses",
+	Long:    `analyse performs full Stack Analyses. Supported Ecosystems are Pypi, Maven, npm and golang.`,
+	Run:     runAnalyse,
+	PostRun: destructor,
```

Comment:
Do the PostRun takes care of ctrl c? i.e. when SA is in progress, and if you cancel it(ctrl + c), will it delete the intermediate files?

Reply:
No, it isn't.

Reply:
file Base is changed to `/tmp`, now if a client hits ctrl c, files is there in tmp directly only.. His workspace will not be polluted

Reply:
Please file a bug to clear the tmp as well. It is okay as of now.

Reply:
Also anti virus might flag this if you leave garbages often in the tmp.

Reply:
`tmp` will be uncleared only we user in any way interrupts the execution. Any successful execution of `analyse` command irrespective of Stack Analyses success or failure, all temporary files will be cleared by our custom destructor.